config/asrockrack.go

```diff
@@ -105,6 +105,15 @@ func (cm *asrockrackVendorConfig) Marshal() (string, error) {
 	}
 }
 
+func (cm *asrockrackVendorConfig) Unmarshal(cfgData string) (err error) {
+	err = xml.Unmarshal([]byte(cfgData), cm.ConfigData)
```

Comment:
would prefer we go with explicit returns 
```suggestion
	return xml.Unmarshal([]byte(cfgData), cm.ConfigData)
```